### ISSUE TITLE
Problem with accessing database on sdcard

### ISSUE THREAD

**Issue by tomikmar** [helloworld1/AnyMemo#423]
After updating system I cannot access database on sdcard.

Steps to reproduce:
* Use "Open" tab - you are in /storage/emulated/0/anymemo
* Use ".." to go to the parent directory - you are in /storage/emulated/0/
* Use ".." to go to the parent directory - message appears: "AnyMemo could not get permission to change to the directory."

The message is correct, but it also means that it is not possible to access database on my sdcard. My rights are like this:
```
  /storage/             drwxr-xr-x  5 root root
  /storage/emulated     drwx--x--x  6 root sdcard_rw (here is missing "r")
  /storage/emulated/0   drwxrwx--x 43 root sdcard_rw
```
Maybe allowing to provide path manually (e.g. "Open path" and "Open default directory " in menu) could be a solution.

Versions:
* AnyMemo 10.9.922 (F-Droid build)
* CyanogenMod 13.0-20161218-NIGHTLY-espresso3g

**Comment by helloworld1** [comment]
Can you just click "anymemo" folder now to access your cards? I tried to repo by clicking ".." several times, I gets stuck on "/storage/emulated/0". But I can still click on "anymemo" folder.

**Comment by tomikmar** [comment]
I wasn't precise in my previous post.  By sdcard I meant real sdcard, not "emulated" one. I need to go to:
* /storage/1421-ED9B/anymemo

**Comment by tomikmar** [comment]
Maybe just adding to AnyMemo options last path (with possibility to change) or default path (which would be used instead of last path if provided) would be quicker to implement. It is not a perfect solution, but it would solve the problem as well.

**Comment by helloworld1** [comment]
Yes, I will try out a solution that is easy to implement and handle the case gracefully.
